examples/templates/azure-linux/cloud-config.yaml.tftpl

```diff
@@ -0,0 +1,58 @@
+#cloud-config
+cloud_final_modules:
+- [scripts-user, always]
+device_aliases:
+  # this is a little fragile because the data volume isn't always sdc when the workspace is stopped
+  # and started.  But, it appears that it is always sdc when the workspace is *first* created.
+  # This allows us to format and label the volume, and then we rely on the label to mount on
+  # subsequent workspace builds.
+  homedir: /dev/sdc
+disk_setup:
+  homedir:
+    table_type: gpt
+    layout: true
+fs_setup:
+  - label: coder_home
+    filesystem: ext4
+    device: homedir.1
+mounts:
+  - ["LABEL=coder_home", "/home/${username}"]
+hostname: ${hostname}
+users:
+  - name: ${username}
+    sudo: ["ALL=(ALL) NOPASSWD:ALL"]
+    groups: sudo
+    shell: /bin/bash
+packages:
+  - git
+write_files:
+  - path: /opt/coder/init
+    permissions: "0755"
+    encoding: b64
+    content: ${init_script}
+  - path: /etc/systemd/system/coder-agent.service
+    permissions: "0644"
+    content: |
+      [Unit]
+      Description=Coder Agent
+      After=network-online.target
+      Wants=network-online.target
+
+      [Service]
+      User=${username}
+      ExecStart=/opt/coder/init
+      Environment=CODER_AGENT_TOKEN=${coder_agent_token}
+      Restart=always
+      RestartSec=10
+      TimeoutStopSec=90
+      KillMode=process
+
+      OOMScoreAdjust=-900
+      SyslogIdentifier=coder-agent
+
+      [Install]
+      WantedBy=multi-user.target
+runcmd:
+  - chown ${username}:${username} /home/${username}
```

Comment:
```suggestion
  - chown -R ${username}:${username} /home/${username}
```

Reply:
I worry this could rewrite ownership in unintended ways, especially around groups